IronPython3Tests/IronPython3Tests.csproj

```diff
@@ -0,0 +1,34 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+
+    <IsPackable>false</IsPackable>
+    <IsTestProject>true</IsTestProject>
+  </PropertyGroup>
+
+  <ItemGroup>
+	  <PackageReference Include="DynamoVisualProgramming.Tests" Version="3.1.0-beta3874">
+		  <PrivateAssets>all</PrivateAssets>
+	  </PackageReference>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.6.0" />
+    <PackageReference Include="NUnit" Version="3.13.3" />
+    <PackageReference Include="NUnit3TestAdapter" Version="4.2.1" />
+    <PackageReference Include="NUnit.Analyzers" Version="3.6.1" />
```

Comment:
nunit adapter, nunit, test.sdk are included by default through DynamoVisualProgramming.Tests

Reply:
I think these were added by visual studio when I started a test project type, I guess if I do remove them then I'll need to mess with the private assets setting below, which I would rather not do - do you have a preference?

Reply:
actually they seem to flow into the bin folder anyway.

Reply:
Were the nunit references added by default ?...
I think the nunit dependencies should be hidden (come from the dynamo deps). It would make it easier to use ..and we could replace them more easily in the future